File: app/src/main/java/com/example/test_task_magents/retrofit/ServiceApi.java
package com.example.test_task_magents.retrofit;

import com.example.test_task_magents.model.GetPictureData;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface ServiceApi {
    @GET("list?")
    Call<List<GetPictureData>> getPicture(@Query("page") int id,
                                          @Query("limit") int limit);
}

File: app/src/main/java/com/example/test_task_magents/retrofit/ServiceApi.kt
package com.example.test_task_magents.retrofit

import com.example.test_task_magents.model.GetPictureData
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ServiceApi {
    @GET("list?")
    fun getPicture(@Query("page") id: Int, @Query("limit") limit: Int): Call<List<GetPictureData>>
}
